ControlRoomApplication/ControlRoomApplication/GUI/AzimuthInputDialog.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using ControlRoomApplication.Constants;
+using ControlRoomApplication.Controllers;
+
+namespace ControlRoomApplication.GUI
+{
+    public partial class AzimuthInputDialog : InputDialog
+    {
+        private double elevationHighLimit;
+        private double elevationLowLimit;
+        private double elevationPos;
+        private double azimuthPos;
+        private Timer timer;    // Timer is responsible for constantly updating the form. 
+        private string[] values;
+        private RadioTelescopeController rtController;
+
+        public AzimuthInputDialog(RadioTelescopeController rtController)
+        {
+            timer = new Timer();
+            timer.Interval = 5;
+            timer.Tick += new EventHandler(timer_Tick);
+            timer.Start();
+
+            this.rtController = rtController;
+
+            InitializeComponent();
+        }
+
+        public double getAzimuthPos()
+        {
+            return azimuthPos;
+        }
+        public double getElevationPos()
+        {
+            return elevationPos;
+        }
+
+        // Handle timer tick events 
+        public void timer_Tick(Object sender, EventArgs e)
+        {
+            System.Diagnostics.Debug.WriteLine("TIMER TICK");
+
+            // Update the limits 
+            if (rtController.EnableSoftwareStops)
+            {
+                elevationHighLimit = rtController.RadioTelescope.maxElevationDegrees;
+                elevationLowLimit = rtController.RadioTelescope.minElevationDegrees;
+            }
+            else
+            {
+                elevationHighLimit = SimulationConstants.LIMIT_HIGH_EL_DEGREES;
+                elevationLowLimit = SimulationConstants.LIMIT_LOW_EL_DEGREES;
+            }
+
+            // Update prompt
+            promptLabel.Text = "The Radio Telescope is currently set to be type " + rtController.RadioTelescope.teleType + "." +
+                            " This script is best run with a telescope type of SLIP_RING.\n\n" +
+                            "Please type an a custom orientation containing azimuth between 0 and 360 degrees," +
+                                " and elevation between " + elevationLowLimit + " and " + elevationHighLimit +
+                                " degrees. Format the entry as a comma-separated list in the format " +
+                                "azimuth, elevation. Ex: 55,80";
+
+            // Disable the OK button if the input is invalid 
```

Comment:
With the timer removed, we can also add a `ToolTip` for our InputDialogs. If you look elsewhere in other forms, you will see that any invalid input will turn the input box yellow and display a `ToolTip`. Basically, whenever the `TextChanged` event goes off, if the input is invalid we can show the `ToolTip` and make the box yellow.

- [ ] Add a `ToolTip` for the input box